xapian-bindings/configure.ac

```diff
@@ -287,11 +291,11 @@ case $with_python$with_python3$with_php7$with_ruby$with_tcl$with_csharp$with_jav
   test -z "$with_ruby" && with_ruby=no
   test -z "$with_perl" && with_perl=no
   test -z "$with_lua" && with_lua=no
+  test -z "$with_go" && with_go=no
   ;;
 esac
 
 BINDINGS=
-
```

Comment:
You've deleted a blank line here - don't reformat existing code as part of a functional change.

Reply:
Not addressed.